homeassistant/components/velbus/__init__.py

```diff
@@ -85,10 +86,13 @@ async def async_setup_entry(hass: HomeAssistant, entry: VelbusConfigEntry) -> bo
         entry.data[CONF_PORT],
         cache_dir=hass.config.path(STORAGE_DIR, f"velbuscache-{entry.entry_id}"),
     )
-    await controller.connect()
+    try:
+        await controller.connect()
+    except VelbusConnectionFailed as error:
+        raise ConfigEntryNotReady("Cannot connect to Velbus") from error
```

Comment:
I think we like to make these translatable now.

Reply:
Yes correct but that is not yet done in the integration, so this would be a complete new part for the velbus integration.